languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/hunspell/create_dict.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+echo "Create morfologik spelling dictionary, based on Hunspell dictionary"
```

Comment:
Have you actually used this to create the dictionary? Did it work the way it is here?